1.5.0 base

robert-nix opened the pull request:
Reversed new stuff, updated the code and the wiki page (https://github.com/Mischanix/sc2replay-csharp/wiki/replay.game.events); all before SC2Gears!

robert-nix committed:
1.4.4 db

robert-nix committed:
1.5.0 base

Some flags will need further inspection (player join got at least 1
extra bitflag, for example); otherwise, works for what little test data
I could gather

ghost commented:
Worth noting the events parser works 100% on 1.5.0 for a small test replay.  Once I can get some larger replays (I made a reddit post in lieu of drop.sc), I'll fix any issues that arise from those immediately.

Also, here's a gist with the SQL changes for the new build data: https://gist.github.com/3228306

Also, 1.5.0 changed the string format to be nicer to L2 caches, so here's the updated dumper code: https://gist.github.com/3228431

robert-nix committed:
Bug fix

Fixed a bug where AbilityEvent.Actors wasn't populated for a
non-rightclick event.  Also tested against a 1.5.0 ladder TvZ
successfully.
